### ISSUE TITLE
Adding sequencing information to biomaterial

### ISSUE THREAD

**Issue by mestato** [statonlab/hardwoods_site#42]
Our biomaterials are usually associated with a set of DNA or RNA sequences (RNA in the case of expression analysis). For an example, if you go look at a biosample in NCBI, it links to experiments and runs (IE sequenced data). We need to import and display that information as well. It has been imported manually to some pages (see tables at bottom of for example the bottom of https://hardwoodgenomics.org/node/68249) but would be better to store in the database and be able to automatically pull from NCBI.

This could be handled by creating biosample props or by creating a new analysis type. Former seems easier, latter seems more Chado/Tripal like. Need further discussion.

**Comment by bradfordcondon** [comment]
both are fine.

Creating a new analysis type amounts to the same thing: creating analysisprops that will hold that information.

**Comment by bradfordcondon** [comment]
> if you go look at a biosample in NCBI, it links to experiments and runs (IE sequenced data).

If this is true for any of our biosamples, please let me know.  Right now it appears to me this is not the case, for example  https://www.ncbi.nlm.nih.gov/biosample/SAMN00167644

Implementing this is super easy: define some props (attached to analysis i guess) that describe the sequencing information.  Then when you create new analyses you just fill out those props.

Getting the loader to automatically pull it in... I'd have to see an example where that information is actually tied to the biosample to comment on that.

I'm adding the "meeting discussion" tag for this because i see no way forward currently and think we should close the issue.

**Comment by bradfordcondon** [comment]
@mestato agreed to close this issue because it doesn't apply to all our biosamples, and its part of a much bigger umbrella that we want to deal with later
